src/main/java/bio/terra/service/dataset/flight/ingest/IngestPopulateFileStateFromFlightMapStep.java

```diff
@@ -51,7 +56,16 @@ public StepResult doSkippableStep(FlightContext context) {
     List<String> errors = new ArrayList<>();
     try (var bulkFileLoadModels = getModelsStream(ingestRequest, fileColumns, errors)) {
 
-      loadService.populateFiles(loadId, bulkFileLoadModels, batchSize);
+      List<FileModel> existingFiles = new ArrayList<>();
```

Comment:
Would you want to use `Set` here since you don't need to keep duplicates?

Also, if this is only used inside the `if`, it would be better to declare it inside the `if` so it has a smaller scope.